src/Widget/WidgetInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Widget;
+
+interface WidgetInterface
+{
+    public function getName(): string;
+
+    public function getType(): string;
+
+    /**
+     * @return string from Bolt\Snippet\Target constants enum
+     */
+    public function getTarget(): string;
+
+    public function getPriority(): int;
+
+    public function getTemplate(): string;
```

Comment:
This does not need to be part of this interface

Reply:
Removed.